traitlets/tests/test_traitlets.py

```diff
@@ -1495,10 +1493,10 @@ class TestDottedObjectName(TraitTestBase):
     obj = DottedObjectNameTrait()
 
     _default_value = "a.b"
-    _good_values = ["A", "y.t", "y765.__repr__", "os.path.join", u"os.path.join"]
-    _bad_values = [1, u"abc.€", "_.@", ".", ".abc", "abc.", ".abc.", None]
+    _good_values = ["A", "y.t", "y765.__repr__", "os.path.join", "os.path.join"]
```

Comment:
double os.path.join now :-)

Reply:
keen eye. i was afraid there'd be a duplicate i would miss.